source/core/src/main/com/csse3200/game/entities/Entity.java

```diff
@@ -129,10 +129,12 @@ public void setScale(Vector2 scale) {
    *
    * @param x width in metres
    * @param y height in metres
+   * @return
    */
-  public void setScale(float x, float y) {
+  public Entity setScale(float x, float y) {
```

Comment:
Why is the entity class being modified? Will this affect other teams?

Reply:
i dont think so its returning null anyway